### PR TITLE
Pricing refactoring

### PR DESCRIPTION
**Pull request by akshay-bharadva**
- [x] created styles - module
- [x] js -> ts refactor
- [x] style refactor
- [x] added standard media breakpoint
- [x] create variables of common styles
- [ ] refactor svgs
- [x] use rem instead of px

### CODE REVIEW

File: src/modules/community-edition/styles.module.css [hunk 1]
```diff
@@ -9,32 +9,37 @@
...
 
 .options .title {
-  font-family: "Work Sans";
+  font-family: var(--ifm-font-family-work-sans);
   font-style: normal;
   font-weight: 700;
   font-size: 2.5rem;
-  line-height: 110%;
+  line-height: var(--ifm-font-line-height);
   letter-spacing: -0.015em;
```
Comment (palashgdev) [inline src/modules/community-edition/styles.module.css]: rem?

File: src/modules/community-edition/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <section className={styles.options}>
+      <div className="container">
+        <h3 className={styles.title}>Community Edition</h3>
+        <p className={styles.subTagline}>
+          Open source version of SigNoz to get started with observability.
+        </p>
+        <Link
+          className={clsx({
+            "button button--primary": true,
+            [styles.readDocsBtn]: true,
```
Comment (palashgdev) [inline src/modules/community-edition/index.tsx]: there is no need of cx if there is no condition

File: src/modules/community-edition/styles.module.css [hunk 3]
```diff
@@ -9,32 +9,37 @@
...
 .options .tagline {
   margin-bottom: 1rem;
-  font-family: "Work Sans";
+  font-family: var(--ifm-font-family-work-sans);
   font-style: normal;
   font-weight: 700;
   font-size: 2rem;
-  line-height: 110%;
+  line-height: var(--ifm-font-line-height);
   letter-spacing: -0.015em;
```
Comment (palashgdev) [inline src/modules/community-edition/styles.module.css]: same here

File: src/modules/community-edition/styles.module.css [hunk 4]
```diff
@@ -9,32 +9,37 @@
...
 }
 .options .subTagline {
   margin-bottom: 1.2rem;
-  font-family: "Work Sans";
+  font-family: var(--ifm-font-family-work-sans);
   font-style: normal;
   font-size: 1.2rem;
-  line-height: 110%;
+  line-height: var(--ifm-font-line-height);
   letter-spacing: -0.015em;
```
Comment (palashgdev) [inline src/modules/community-edition/styles.module.css]: here also

File: src/modules/community-edition/styles.module.css [hunk 5]
```diff
@@ -9,32 +9,37 @@
...
-  color: #ffffff;
+  color: var(--ifm-color-white);
+}
+.options .readDocsBtn {
+  margin: 6px;
+  padding-left: 10px !important;
+  padding-right: 10px !important;
```
Comment (palashgdev) [inline src/modules/community-edition/styles.module.css]: rem

File: src/modules/community-edition/styles.module.css [hunk 6]
```diff
@@ -9,32 +9,37 @@
...
   font-style: normal;
   font-size: 1.2rem;
-  line-height: 110%;
+  line-height: var(--ifm-font-line-height);
   letter-spacing: -0.015em;
-  color: #ffffff;
+  color: var(--ifm-color-white);
+}
+.options .readDocsBtn {
+  margin: 6px;
```
Comment (palashgdev) [inline src/modules/community-edition/styles.module.css]: here also

File: src/modules/community-edition/styles.module.css [hunk 7]
```diff
@@ -50,19 +55,19 @@
   margin: 0 1rem 1rem;
   list-style-type: none;
 }
-@media (max-width: 996px) {
-}
-@media (max-width: 568px) {
+
+@media (max-width: 768px) {
```
Comment (palashgdev) [inline src/modules/community-edition/styles.module.css:57]: token for 768?

File: src/modules/try-signoz-cta/index.tsx [hunk 8]
```diff
@@ -16,7 +18,11 @@ export const TrySigNozCTA = () => {
         <div className={styles.contentWrapper}>
           <h3 className={styles.tagline}>Give SigNoz a try</h3>
           <a
-            className="button button--secondary"
+            className={clsx({
```
Comment (palashgdev) [inline src/modules/try-signoz-cta/index.tsx]: same here also

File: src/modules/why-opentelemetry/styles.module.css [hunk 9]
```diff
@@ -1,8 +1,8 @@
...
+  color: var(--ifm-color-primary);
+}
 
-.iconImage{
-    height: 120px;
-    width: 120px;
-}
\ No newline at end of file
+.iconImage {
+  height: 120px;
```
Comment (palashgdev) [inline src/modules/why-opentelemetry/styles.module.css]: rem

File: src/pages/pricing/index.js [hunk 10]
```diff
@@ -24,30 +26,49 @@ const handleClick = (message) => {
...
   return (
     <Layout title="SigNoz Plans">
-      <section className="pricing">
+      <section className={styles.pricing}>
         <div
-          className="container"
-          style={{ marginTop: "3rem", marginBottom: "4rem" }}
+          className={clsx({
+            container: true,
+            [styles.pricingContainer]: true,
```
Comment (palashgdev) [inline src/pages/pricing/index.js]: here also

File: src/pages/pricing/index.js [hunk 11]
```diff
@@ -376,14 +458,24 @@ function pricingTest() {
...
+          className={clsx({
+            container: true,
+            [styles.faqContainer]: true,
+          })}
         >
           <div className="row">
             <div className="col col--8 col--offset-2">
-              <p className="hero__subtitle margin--md title">FAQs</p>
+              <p
+                className={clsx({
```
Comment (palashgdev) [inline src/pages/pricing/index.js]: check every where

File: src/pages/pricing/styles.module.css [hunk 12]
```diff
@@ -0,0 +1,202 @@
+/* pricing */
+
+.pricing * {
+  font-family: var(--ifm-font-family-work-sans);
+}
+.pricing hr {
+  margin: 1.2rem 0;
+  border: 0.5px solid #464646 !important;
```
Comment (palashgdev) [inline src/pages/pricing/styles.module.css]: token for color ?

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: src/css/custom.css [hunk 13]
```diff
@@ -6,21 +6,56 @@
...
+  /* FONT */
+  --ifm-code-font-size: 95%;
+  --ifm-font-line-height: 110%;
   /* --ifm-font-family-base: "Space Grotesk"; */
   --ifm-font-family-base: arial;
   --ifm-font-size-base: 14pt;
+  --ifm-font-family-work-sans: "Work Sans", sans-serif;
+  --ifm-font-letter-spacing: -0.015rem;
+  /* MEDIA BREAKPOINT */
+  --ifm-mobile-start: 320px;
```
Comment (palashgdev) [inline src/css/custom.css]: please remove this one if we are not using this one

File: src/modules/signup-form/index.tsx [hunk 14]
```diff
@@ -177,9 +174,7 @@ export default function SignUpFormReact() {
     return (
       <button
         type="submit"
-        className={clsx({
-          [styles.submitBtn]: true,
-        })}
+        className={`${styles.submitBtn}`}
```
Comment (palashgdev) [inline src/modules/signup-form/index.tsx]:
```suggestion
        className={styles.submitBtn}
```

File: src/pages/pricing/index.js [hunk 15]
```diff
@@ -3,18 +3,18 @@ import Layout from "@theme/Layout";
...
+import { RightSVG, WrongSVG } from "../../svgs/common";
+import { TalkToExpert } from "../../modules/talk-to-expert";
+import { CommunityEdition } from "../../modules/community-edition";
+import { UserReview } from "../../modules/user-review";
+import { TrySigNozCTA } from "../../modules/try-signoz-cta";
+import { DataProtection } from "../../modules/data-protection";
+import styles from "./styles.module.css";
 
-// ReactGA.initialize("UA-152867655-1");
+ReactGA.initialize("UA-152867655-1");
```
Comment (palashgdev) [inline src/pages/pricing/index.js:15]: why we are initialising here ?

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: src/pages/pricing/index.js (hunk 15 above)
Comment (akshay-bharadva) [reply src/pages/pricing/index.js:15]: @palashgdev
This change will be removed once I create form for `get-cloud` and `self-hosted` pages

**Pull request merged**